.travis.yml

```
@@ -84,7 +84,7 @@ jobs:
         # prepare deployment envs
         - source .gravitybee/gravitybee-environs.sh
         - export SATS_TAG=$GB_ENV_APP_VERSION
-        - export SATS_BODY="* [$GB_ENV_APP_NAME v$GB_ENV_APP_VERSION CHANGELOG](https://github.com/plus3it/$GB_ENV_APP_NAME/blob/$GB_ENV_APP_VERSION/CHANGELOG.md)"
+        - export SATS_BODY="$(echo -e '* ['$GB_ENV_APP_NAME' v'$GB_ENV_APP_VERSION' CHANGELOG](https://github.com/plus3it/'$GB_ENV_APP_NAME'/blob/'$GB_ENV_APP_VERSION'/CHANGELOG.md)\n* ['$GB_ENV_APP_NAME' v'$GB_ENV_APP_VERSION' SCAP REPORTS](https://watchmaker.cloudarmor.io/list.html#releases/'$GB_ENV_APP_VERSION'/scans)')"
```

End the link with a `/` to automatically open the directory, instead of just browse to it... E.g.

* https://watchmaker.dev.cloudarmor.io/list.html#releases/0.23.2/scans/

vs

* https://watchmaker.dev.cloudarmor.io/list.html#releases/0.23.2/scans

```suggestion
        - export SATS_BODY="$(echo -e '* ['$GB_ENV_APP_NAME' v'$GB_ENV_APP_VERSION' CHANGELOG](https://github.com/plus3it/'$GB_ENV_APP_NAME'/blob/'$GB_ENV_APP_VERSION'/CHANGELOG.md)\n* ['$GB_ENV_APP_NAME' v'$GB_ENV_APP_VERSION' SCAP REPORTS](https://watchmaker.cloudarmor.io/list.html#releases/'$GB_ENV_APP_VERSION'/scans/)')"
```

---

Ok.  Updated.